website/contribute/rfc-process.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 3
+title: "RFC Process"
+toc: true
+last_modified_at: 2020-09-01T15:59:57-04:00
+---
+
+This page describes a Request For Comments (RFC) process for proposing any major change to Hudi or even just sharing designs/vision for the project to get early feedback.
+
+## When do you need an RFC ?
+
+Whenever a feature is considered to be causing a “major change” in the project, such a feature requires an RFC. 
+Any of the following can be considered a major change:
+
+ - Any new component, module or code that introduces a new concept into the project or alters the behavior of an existing one
+ - Any large code refactor to address general code re-usability and structure. There is no strong definition for “Large” and whether or not the refactor requires an RFC can be discussed on the @dev mailing list.
+ - Any change that impacts the underlying storage layout/format. e.g changes to the HoodieLogFormat, timeline layout version.
+ - New indexing schemes, New deltastreamer sources, New platform services.
+ - Any change that impacts the public interfaces of the project.
+
+It can also be used to describe large direction shifts to the project (e.g: Flink support) or new green field ideas (e.g Hudi for ML flows)
```

Comment:
```suggestion
It can also be used to describe large direction shifts to the project (e.g.: Flink support) or new green field ideas (e.g. Hudi for ML flows)
```